lib/mappers/details.js

```diff
@@ -129,17 +129,24 @@ function buildHistogram (container) {
   };
 }
 
-function extractComments (comments) {
-  if (!comments) {
+// Google Play responds with the country us and lang en
```

Comment:
Have you verified this by code or just in the browser? I've noticed that when logged in the browser the mappings differ, could that be related?

I find it suspicious that this only happens for this field, I wonder if there are other scenarios we're missing

Reply:
I looked into google play on an anonymous tab, and `ds:17` is the only field that has the `isError` attributte set to true, and **only** when I add `hl=en&gl=us` as url parameters.
```javascript
<script nonce="">AF_initDataCallback({key: 'ds:17', isError:  true ,  data:["er"]});</script>
```

When I logged in google play, the only field with `isError` true is `ds:24`:
```javascript
<script nonce="">AF_initDataCallback({key: 'ds:24', isError:  true ,  data:["er"]});</script>
```

The mappings for comments also change, `ds:27` for `us/en` and `ds:25` for any other country/lang